### PR TITLE
{platform}.Syntax modules

### PR DESCRIPTION
**Pull request by RealyUniqueName**
Unified some methods in modules:
php.Syntax,
js.Syntax,
python.Syntax

@nadako, please, review js and python related changes

### CODE REVIEW

File: std/js/Syntax.hx [hunk 1]
```diff
@@ -54,4 +54,11 @@ extern class Syntax {
 	**/
 	@:overload(function(o:Dynamic, f:Int):Bool {})
 	static function delete(o:Dynamic, f:String):Bool;
+
+	/**
+		Generate `o[f]` expression
+	*/
+	static inline function field(o:Dynamic, f:String):Dynamic {
+		return code('{0}[{1}]', o, f);
```
Comment (nadako) [inline std/js/Syntax.hx]: I would expect a method named `Syntax.field` generate `o.f`, not `o[f]`.

File: std/js/_std/HxOverrides.hx [hunk 2]
```diff
@@ -40,7 +40,7 @@ class HxOverrides {
 		switch( s.length ) {
 		case 8: // hh:mm:ss
 			var k = s.split(":");
-			var d = js.Syntax.new_(Date);
+			var d = js.Syntax.construct(Date);
 			(cast d)[cast "setTime"](0);
```
Comment (nadako) [inline std/js/_std/HxOverrides.hx]: I wonder why we do array access here instead of simply `d.setTime`...

Comment (RealyUniqueName) [reply std/js/_std/HxOverrides.hx]: That's not in the scope of this PR :)

File: std/js/Syntax.hx (hunk 1 above)
Comment (RealyUniqueName) [reply std/js/Syntax.hx]: I failed to find a better name.
Also it mimics well known `Reflect.field()` method.

Comment (kaikoga) [reply std/js/Syntax.hx]: Perhaps you meant `o["f"]` , in case of something like `o["class"]` becoming `o.class`?

Comment (RealyUniqueName) [reply std/js/Syntax.hx]: `f` is an identifier here, so i meant `o[f]` without quotes.
Example:
```haxe
var fieldName:String = getFieldName();
Syntax.field(obj, fieldName); //generates obj[fieldName]
```

**Comment by RealyUniqueName** [comment]
@nadako Can I merge this?

**Comment by nadako** [comment]
yeah, lgtm

**Pull request merged**